### DISCUSSION TITLE
some error about rabbitmq

### DISCUSSION THREAD

**Discussion by loveyang2012** [rabbitmq/rabbitmq-server#7348] [Other]
I have gotten some error about rabbitmq(3.8.14), and my app does not work very well.
some error look like these:
![image](https://user-images.githubusercontent.com/24237714/219545219-675f2d90-7f14-45a4-b483-d6f5611c2c8f.png)
![image](https://user-images.githubusercontent.com/24237714/219545279-41672f99-afa2-49fc-81bb-fa284a0eaeb5.png)
does it matter ? 
thanks.

**Comment by michaelklishin** [comment]
A channel on a closing connection had run into an exception. You can ignore this as log noise.

RabbitMQ 3.8 has been [completely out of support since July 2022](https://rabbitmq.com/versions.html). Consider [upgrading](https://rabbitmq.com/changelog.html) to 3.11.x.
